src/ripple/rpc/handlers/DownloadShard.cpp

```diff
@@ -34,7 +34,6 @@ namespace ripple {
 /** RPC command that downloads and import shard archives.
     {
       shards: [{index: <integer>, url: <string>}]
-      validate: <bool> // optional, default is true
```

Comment:
Just wondering if this would be considered an API break?  Or maybe we don't worry about it because Shards are not really a released feature yet?

Reply:
Its an optional field and will be silently ignored if set. It is important to also note that is an admin only command. The drawback is that the import process will be slower for admins attempting to set the field to false as the data will always be validated. The change has brought to Rome's attention and he will update documentation accordingly.